src/tests/receiving/editBinLocationWhenReceive.test.ts

```diff
@@ -56,6 +51,7 @@ test.describe('Edit Bin Location when receive inbound stock movement', () => {
         ).toBeVisible();
         await locationListPage.getLocationEditButton(mainLocation.name).click();
         await createLocationPage.binLocationTab.click();
+        await page.waitForTimeout(1000);
```

Comment:
Please add a comment explaining why this is here. Otherwise, if it is not required, then remove (same for the timeouts below). Perhaps if we are waiting for something to load here eitherway - in the next line, then there would be an option to change and try to wait for another element to load first instead of adding this timeout?)

Reply:
I've changed the usage of timeouts.